### PR TITLE
chore: added Fontesk to frontend database under subcategory fonts

### LINKED ISSUE #562: [Add] Fontesk
### Site URL

https://fontesk.com/

### Description of the above provided site

A curated font library of the best high-quality free fonts to download for commercial and personal use. The site uses renderings to showcase visualizations of the font in a curated design.

### Category

Frontend

### Sub Category

fonts

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by jjxrry**
## Fixes Issue

Closes #562

## Changes proposed

Added Fontesk to frontend database under subcategory fonts
 

## Screenshots

Located under "fonts" in the "frontend" folder
8acd88734246c979aa252889f87899f5

### CODE REVIEW

File: database/frontend/fonts.json [hunk 1]
```diff
@@ -32,5 +32,12 @@
         "url": "https://type-scale.com",
         "category": "frontend",
         "subcategory": "fonts"
+    },
+    {
+        "name": "Fontesk",
+        "description": "Curated web platform font library of the best high quality free fonts to download for commercial and personal use.",
```
Comment (CBID2) [inline database/frontend/fonts.json:38]: [Proofreading] The description was a bit unclear due to its use of the preposition "of". 
```suggestion
        "description": "Curated web platform font library that contains the best high-quality free fonts to download for commercial and personal use.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @jjxrry! :) You're off to a great start! :) There were a few typos in the description, so refer to the inline comment to fix them! :) Keep it up! :)

**Review by rupali-codes — DISMISSED** [review]
looks good

**Comment by rupali-codes** [comment]
@jjxrry are you a gssoc contributor?

**Comment by jjxrry** [comment]
@rupali-codes no unfortunately I did not make this year's application deadline as I found out about it recently.

**Comment by jjxrry** [comment]
@rupali-codes are you referencing the announcement you made in the discord? if so, can anyone contribute as I would love to be involved

**Comment by CBID2** [comment]
> @rupali-codes are you referencing the announcement you made in the discord? if so, can anyone contribute I would love to be involved.
Yes that's the one @jjxrry. If you are in GSSOC, you'd have to check the GSOC checkmark when you raise an issue(see screenshot below)

GSSOC check mark

**Review by jjxrry — COMMENTED** [review]
fixed grammar for Fontesk JSON resource

**Comment by rupali-codes** [comment]
> @rupali-codes are you referencing the announcement you made in the discord? if so, can anyone contribute as I would love to be involved

I'm just confirming so we can add the _gssoc23_ label :)

File: database/frontend/fonts.json [hunk 2]
```diff
@@ -32,5 +32,12 @@
         "url": "https://type-scale.com",
         "category": "frontend",
         "subcategory": "fonts"
+    },
+    {
+        "name": "Fontesk",
+        "description": "Curated web platform font library that contains the best high-quality free fonts to download for commercial and personal use.",
```
Comment (CBID2) [inline database/frontend/fonts.json:38]: [Proofreading]
```suggestion
        "description": "Curated online font library that contains the best high-quality free fonts to download for commercial and personal use.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @jjxrry! :) I pointed out a proofreading error that needs the be fixed.

**Comment by rupali-codes** [comment]
@jjxrry could you please make the suggested changes?

**Comment by jjxrry** [comment]
apologies for the delay, just made changes and requested reviews

**Review by rupali-codes — APPROVED** [review]
looks good to me

**Pull request merged**